src/Methods/Chat.php

```diff
@@ -73,9 +71,7 @@ public function kickChatMember(array $params): bool
      */
     public function leaveChat(array $params): bool
     {
-        $this->get('leaveChat', $params);
-
-        return true;
+        return $this->get('leaveChat', $params)->getResult();
```

Comment:
Initially I thought the result was coming back as a string. I checked again and it is a bool so I've removed the cast on the others.. Good catch.